<?xml version="1.0" ?>
<launch>
    <param name="/use_sim_time" value="true" />

    <!-- in maize, static transform between base_footprint and base_link was missing -->
    <node pkg="tf" type="static_transform_publisher" name="odom_T" args="0.0 0.0 0.05 0.0 0.0 0.0 1.0 /base_footprint /base_link 100"/>

     <group ns="muse_mcl_2d_mapping_3d_ocm_node">
     <rosparam param="lasers2d">
     </rosparam>
     <rosparam param="lasers3d">
         ["/velodyne_points"]
     </rosparam>
     <param name="subscriber_queue_size" value="10" />
     <param name="rate" value="0.0" />
     <param name="resolution" value="0.05" />
     <param name="map_frame" value="/odom" />
     <param name="ndt_map_pub_rate" value="10.0" />
     <param name="occ_map_pub_rate" value="10.0" />
     <param name="ndt_sampling_resolution" value="0.25" />
     <param name="ndt_grid_resolution" value="2.0" />
    </group>

    <node name="muse_mcl_2d_mapping_3d_ocm_node" pkg="muse_mcl_2d_mapping" type="muse_mcl_2d_mapping_3d_ocm_node" output="screen" />

</launch>
